### PR TITLE
[EXTREME TESTING REQUIRED] Facebook authentication error 

### PR DESCRIPTION
**Pull request by Krakaw**
### References Issues:
Fixes the false negatives for facebook auth.
### Environment Variables
 * No change

### CODE REVIEW

File: src/components/pages/authentication/social/FacebookButton.js [hunk 1]
```diff
@@ -115,11 +130,16 @@ class FacebookButtonDisplay extends Component {
 					});
 				});
 		} else {
-			notifications.show({
-				message: "Facebook authentication failed.",
-				variant: "error"
-			});
-			this.setState({ isAuthenticating: false });
+			if (status !== "unknown") {
```
Comment (Jasonvdb) [inline src/components/pages/authentication/social/FacebookButton.js]: If the status is unknown then won't the user just never see it failed?

Comment (Krakaw) [reply src/components/pages/authentication/social/FacebookButton.js]: The status is unknown if you didn't login with facebook and there is a null authResponse

**Comment by Krakaw** [comment]
You can replicate this error by logging in as a user, then manually change the url to http://localhost:3000/login

**Comment by Jasonvdb** [comment]
Cool will test properly

**Comment by Jasonvdb** [comment]
Tested this, all seems good

**Pull request merged**